liota/lib/utilities/utility.py

```diff
@@ -1,4 +1,3 @@
-
 # -*- coding: utf-8 -*-
```

Comment:
Raise an issue to split and rename this file. A collection of unrelated functions in "lib/utilities/utility" does not look good.